### ISSUE TITLE
Twitch: Change `group` to `videos`, remove `item`

### ISSUE THREAD

**Issue by moollaza** [duckduckgo/zeroclickinfo-spice#1871]
We now have a `videos` template group. The Twitch Featured and Twitch Streams IA's should now be updated to look like so:

```js
templates: {
    group: 'videos',
    detail: false,
    item_detail: false
    // note: we don't need to specify item: 'videos_item'
}
```

**Comment by Dwaligon** [comment]
Can do @moollaza :+1:

**Comment by moollaza** [comment]
Fixed in #1872 -- thanks a lot @Dwaligon !
